### PR TITLE
BUG FIX: Fix js error

### PR DESCRIPTION
**Pull request by ajwl**
This fixes 2 bugs introduced by: https://github.com/guardian/frontend/pull/19437

**1. JS Error when no campaigns were present to a page**
Even if no campaigns were available the render went ahead anyway and threw a js error when no data was found
I've added a check to see if any campaigns are present. If they are not, the process stops. 

**2. Js Error in form submission**
The selector being used to add the form submit event listener to the form was broken. This fixes it, while making it more specific so it doesn't get tangled up.

### CODE REVIEW

**Comment by ajwl** [comment]
NB:
I can't successfully stub out a dependency in one of the tests, now I've changed the code. 
Have followed this guide, but it doesn't seem to work. https://facebook.github.io/jest/docs/en/mock-functions.html#mock-implementations
Any ideas?

```
jest.mock('journalism/modules/get-campaign');
import getCampaign from 'journalism/modules/get-campaign';
getCampaign.mockImplementation(() => {title: 'A great campaign'});
```

**Review by regiskuckaertz — APPROVED** [review]
Small changes as discussed but otherwise, great detective work 🕵️‍♀️

**Comment by ajwl** [comment]
@regiskuckaertz changes made. Thanks

File: static/src/javascripts/projects/journalism/modules/render-campaign.js [hunk 1]
```diff
@@ -16,16 +16,21 @@ const renderCampaign = (anchorNode: HTMLElement): void => {
...
 };
 
 export const initCampaign = () => {
+    const calloutData = getCampaign();
+
     const fourthParagraph = document.querySelector(
         '.content__article-body p:nth-of-type(4)'
     );
-    if (fourthParagraph) renderCampaign(fourthParagraph);
+    if (calloutData && fourthParagraph)
```
Comment (hosamaly) [inline static/src/javascripts/projects/journalism/modules/render-campaign.js]: Does this mean that the campaign shouldn't be rendered if there are 3 paragraphs and a figure?

Comment (ajwl) [reply static/src/javascripts/projects/journalism/modules/render-campaign.js]: For now yes. Though good point. 
The logic for the location on the page is likely to change to place it the end of the story, and this is one of the reasons why.

**Pull request merged**